CONTRIBUTING.rst

```diff
@@ -56,15 +56,15 @@ Now you should be able to ::
     cd PyAbel/doc/
     make livehtml
 
-which should launch a browser window displaying the docs. When you save a change to any of the docs, the re-build should happen automatically and the docs should update in a matter of a few seconds. 
+which should launch a browser window displaying the docs. When you save a change to any of the docs, the re-build should happen automatically and the docs should update in a matter of a few seconds.
 
 Alternatively, `restview <https://pypi.python.org/pypi/restview>`_ is a nice way to preview the ``.rst`` files.
 
 Before merging
 --------------
 
 If possible, before merging your pull request please rebase your fork on the last master on PyAbel. This could be done `as explained in this post <https://stackoverflow.com/questions/7244321/how-to-update-a-github-forked-repository>`_ ::
-   
+
     # Add the remote, call it "upstream" (only the fist time)
     git remote add upstream git@github.com:PyAbel/PyAbel.git
```

Comment:
I may be alone with this, but I have found that only the address given on the home page works:
`https://github.com/PyAbel/PyAbel.git`